src/reflect/scala/reflect/internal/pickling/UnPickler.scala

```diff
@@ -701,6 +705,7 @@ abstract class UnPickler {
       private val definedAtRunId = currentRunId
       private val p = phase
       protected def completeInternal(sym: Symbol) : Unit = try {
+        lazyCompletingSymbol = sym
```

Comment:
I actually tried to reproduce this several times on tests but I did not succeed. Can you provide an example where this would happen?